packages/flutter/lib/src/gestures/multitap.dart

```diff
@@ -16,8 +16,21 @@ import 'tap.dart';
 
 /// Signature for callback when the user has tapped the screen at the same
 /// location twice in quick succession.
+///
+/// See also:
+///
+///  * [GestureDetector.onDoubleTap], which matches this signature.
 typedef GestureDoubleTapCallback = void Function();
 
+/// Signature for callback when the user has tapped the screen at the same
```

Comment:
This description needs to be updated (it's identical to the GestureDoubleTapCallback).